src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPage.java

```diff
@@ -339,14 +477,23 @@ private boolean checkIfPrivateMessagesNotAllowed(String userName) {
     return isElementOnPage(allowPrivateMassageButton);
   }
 
-  private WebElement getElementForUser(String userName, String selector) {
-    String userCss = String.format(selector, userName);
+  private WebElement getElementForUser(String userName) {
+    WebElement kk = (WebElement) By.cssSelector(String.format(PRIVATE_MESSAGE_USER_SELECTOR, userName));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "kk" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)